source/AI.cpp

```diff
@@ -2904,7 +2920,7 @@ Point AI::TargetAim(const Ship &ship, const Body &target)
 		result += p.Unit() * abs(damage);
 	}
 
-	return result ? result : target.Position() - ship.Position();
+	return result ? result : ship.Position() - target.Position();
```

Comment:
Why's this flipped? (Not saying it's wrong, just curious.)

Reply:
~~No idea I didnt do this~~

Reply:
I'll dig into it and figure out why it might have been flipped. My first guess is that normally we get a `result` and so that gets used and we don't notice any odd behavior, but in the cases where we have no `result`, `target - ship` might be the wrong direction to aim.

Reply:
I have seen AI use meteors aimed at the other direction tbf
~~but that bug was reported within this PR too so idk what to think~~

Reply:
That seems like a separate issue: #6024

Reply:
I just did a quick testing with a ship using only seekers and it was able to aim correctly, so idk, mb its because Photon wanted the AI to aim the opposite of the ships in general though

Reply:
> ~~No idea I didnt do this~~

The changes to this line were actually all on you. :james:

Commit from December 21st:
> fix logic issue? ships would aim at the opposite of where the target is and not at the target

Reply:
The comment on the TargetAim function above this one:
> // Get a vector giving the direction this ship should aim in in order to do
> // maximum damaged to a target at the given position with its non-turret,
> // non-homing weapons. If the ship has no non-homing weapons, this just
> // returns the direction to the target.

Reply:
Oh I remember now, I changed it when smbd told me of the meteor bug, thinking it could fix it xD